src/components/subscriptions/constants.js

```diff
@@ -19,5 +19,5 @@ export const SUBSCRIPTION_USERS_OVERVIEW = 'Subscription Users Overview';
 export const NETWORK_ERROR_MESSAGE = 'Error occurred while loading the data.';
 export const DEFAULT_PAGE = 1;
 
-// used to determine whether to show the revocation cap messaging
-export const SHOW_REVOCATION_CAP_PERCENT = 10;
+// used to determine whether to show the revocation cap messaging in the license revoke modal
+export const SHOW_REVOCATION_CAP_PERCENT = 80;
```

Comment:
After conversation with Joe, we're opting to raise this percent to extend number of revocations the customer must do before they would see the revocation cap alert.